x/stakeibc/keeper/msg_server_instant_redeem_stake_test.go

```diff
@@ -0,0 +1,201 @@
+package keeper_test
+
+import (
+	sdkmath "cosmossdk.io/math"
+	"fmt"
+	epochtypes "github.com/Stride-Labs/stride/v5/x/epochs/types"
+	recordtypes "github.com/Stride-Labs/stride/v5/x/records/types"
+	stakeibctypes "github.com/Stride-Labs/stride/v5/x/stakeibc/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	_ "github.com/stretchr/testify/suite"
+)
+
+type InstantRedeemStakeState struct {
+	epochNumber         uint64
+	depositRecordAmount sdkmath.Int
+	initialAmount       sdkmath.Int
+	hostZone            stakeibctypes.HostZone
```

Comment:
I dont think you need `epochNumber` or `initialAmount` here since they're not leveraged directly by any of the tests

Reply:
Yeah, currently true for epochNumber, but initialAmount is used in a test.

Reply:
@sampocs this is addressed. ready to resolve